bitcoin-magazine/bitcoin-explained/bitcoin-core-22.0-explained.md

```diff
@@ -0,0 +1,867 @@
+Sometimes you need to get a transaction unstuck, but that transaction doesn't even make it into the mempool, which means you also can't get it unstuck with child pays for parent.
+And then The solution there would be to have the network to broadcast packages of transactions over a network.
+So in the same way that a package of transactions can sort of be confirmed by a miner all at once because the miner calculates the total fee, that sort of same calculation, the total fee calculation, would come to apply for what regular nodes put in the mempool.
+Yeah.
+That's sort of the long-term upgrade that's being worked on, and this is a very small step in that process, right?
+
+Sjors: 00:29:17
+
+Yeah, and for Lightning, it's a way to reduce fees, right?
+Because right now when you, well, to reduce fees in some cases.
+So right now with Lightning, you have to pre-sign a transaction, which means you have to decide in advance what the fee is going to be for that transaction.
+And so these fees have to be taken some sort of worst case scenario into account.
+Whereas with this change, eventually the hope is that you just set no fee on it, on the conflicting, on the backup transaction that you have in Lightning, and you just send it on the blockchain and then some, you know, whoever really needs this transaction will just pay the extra fee that is needed right then as a child pays for a parent.
+So it's a useful feature, but yeah, there's some problems with relay when the mempool is very full.
+So yeah.
+
+Aaron: 00:30:07
+
+Got it.
+Yeah, that makes sense.
+So this test mempool accept is a small step in improving that process.
+Yep.
+Another incremental improvement.
+
+Sjors: 00:30:17
+
+Exactly.
+
+Aaron: 00:30:18
+
+Alright, So one more to go, Sjors.
+
+Sjors: 00:30:21
+
+Maybe two actually.
+I have some other amount.
+One small one is add multi-sig address.
+I think we talked about that.
+Well, basically the multi-signature support inside the RPC.
+You can now use it for 20 public keys instead of 16.
+This is...
+Which is something that SegWit supports.
+
+Aaron: 00:30:43
+
+Right.
+This is for Bitcore Core users that want to receive a transaction, right now this is limited to, you can create a multi-sig, but it's limited to 16 public keys, is that right?
+
+Sjors: 00:30:57
+
+Yeah, and I think the reason is that the legacy addresses could only do 16 P2SH and SegWit can do 20, but the Bitcoin Core couldn't support it yet.
+
+Aaron: 00:31:08
+
+Right.
+So now Bitcoin Core users can receive transactions on 20 key multisigs.
+
+Sjors: 00:31:15
+
+Yeah, it's not a consensus change.
+None of the rules changed.
+It's just that the Bitcoin Core wallet can now do a little bit more.
+
+Aaron: 00:31:21
+
+Yeah, it was always possible according to the consensus rules, but not according to Bitcoin Core yet, until now.
+
+Sjors: 00:31:27
+
+Yeah.
+So another thing we can mention that is, I think, fun
+
+## Support for NAT-PMP
+
+is support for NAT-PMP.
```

Comment:
As observed here, there are instances where chapter titles occasionally disrupt the text flow (fortunately, this has been addressed in newer AI-generated transcripts). Furthermore, there are instances of misattributed words from one speaker to another, affecting sentence coherence. Ensuring sentences maintain the correct structure is crucial for optimal readability.